proj/src/components/Header.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import moment from 'moment'
+export default (props) => {
+  return (
+    <div className="header-container">
+      <h1 className="title">{props.children}</h1>
+      <h3 className="dates">{moment(props.times[0]).format('MMM DD, YYYY')} - {moment(props.times[6]).format('MMM DD, YYYY')}</h3>
```

Comment:
please extract these to variables and you can also use lodash/fp for safe access of first and last elements in times array. i.e 
`
const endDate = last(times);
const startDate = head(times);
`